config.yaml

```diff
@@ -6,7 +6,12 @@ endpoints:
     conditions:
       - "[STATUS] == 200"
       - "[BODY].status == UP"
-      - "[RESPONSE_TIME] < 150"
+    ui:
+      response-timer-badge-awesome: 550
+      response-timer-badge-great: 850
+      response-timer-badge-good: 1350
+      response-timer-badge-passable: 1650
+      response-timer-badge-bad: 1750
```

Comment:
Does it really make sense for this to be under an endpoint configuration instead of the root `ui` configuration?

Reply:
Hi, we worked together on that change. 
The reason to configure this within the single endpoint was, that every endpoint could have other values. Some endpoints are fine with the default values and some have special needs in terms of an "awesome" response time. So we decided to det the values per endpoint.

Reply:
Makes sense!

Regardless, I'd rather have a configuration that looks like this:
```yaml
    ui:
      badge:
        response-time:
          thresholds: [550, 850, 1350, 1650, 1750]
```

It's not as intuitive/readable as the solution you've suggested, however, it makes it more future proof by making it easier to add more parameters under a single type of badge, e.g.:
```yaml
    ui:
      badge:
        response-time:
          thresholds: [550, 850, 1350, 1650, 1750]
          theme: modern
```
and it also makes it take a little less space consuming if we were to add a similar feature for a different badge in the future, e.g.:
```yaml
    ui:
      badge:
        response-time:
          thresholds: [550, 850, 1350, 1650, 1750]
        uptime:
          thresholds: [99.9, 99, 95, 90, 75]
```